src/rpc/net.cpp

```diff
@@ -129,8 +129,8 @@ static RPCHelpMan getpeerinfo()
                     {RPCResult::Type::STR, "addrbind", /*optional=*/true, "(ip:port) Bind address of the connection to the peer"},
                     {RPCResult::Type::STR, "addrlocal", /*optional=*/true, "(ip:port) Local address as reported by the peer"},
                     {RPCResult::Type::STR, "network", "Network (" + Join(GetNetworkNames(/*append_unroutable=*/true), ", ") + ")"},
-                    {RPCResult::Type::NUM, "mapped_as", /*optional=*/true, "The AS in the BGP route to the peer used for diversifying\n"
-                                                        "peer selection (only available if the asmap config flag is set)"},
+                    {RPCResult::Type::NUM, "mapped_as", /*optional=*/true, "Mapped AS (Autonomous System) number in the BGP route to the peer, used for diversifying\n"
```

Comment:
I agree that the wording isn't perfect, but I still think this can be merged as is.

Reply:
"at the end of the BGP route to the peer" in all of them (including -netinfo as well) seems good

Reply:
agree that that's clearer; it may not always be (due to ASmap being an approximation, as well as incomplete information) but the AS number is supposed ot be the end-point AS, not just one along the route

Reply:
I agree, but since we have some ACKs here, I can leave it for another PR or in case I need to touch it again.

Reply:
yes i am a bit divide about this, i understand you don't want to invalidate ACKs last-minute, but on the other hand re-reviewing after a trivial change is super easy, and if we're touching this anyway it's good to make sure it's the best possible documentation

Reply:
No problem, so I'm gonna address it here.

Reply:
Done.